README.md

```diff
@@ -84,5 +127,16 @@ the necessary settings discussed in the configuration section.
 source my_config
 cd /srv/metadataproxy
 source venv/bin/activate
-gunicorn wsgi:app --workers=2 -k gevent
+gunicorn metadataproxy:app --workers=2 -k gevent
+```
+
+## Run metadataproxy with docker
+
+For production purposes, you'll want to kick up a container to run.
+You can build one with the included Dockerfile.  To run, do something like:
+```bash
+docker run --net=host \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -e MOCK_API=true \
```

Comment:
No reason to MOCK_API in production. MOCK_API is mostly meant for using metadataproxy outside of AWS (like in a vagrant VM). metadataproxy will still mock out the IAM routes, but it'll pass everything else through to the real metadata service.